internal-docs/engineering/sli-slo/readme.md

```diff
@@ -0,0 +1 @@
+# SLIs, SLOs and Monitors
```

Comment:
Question: Is this intended to be fleshed out in this PR? If not, should we just remove it for now?

Reply:
This particular page? Yeah, can be removed. I'm not against placeholders as interim things either